.github/release-drafter.yml

```
@@ -26,3 +26,25 @@ categories:
       - 'test flakiness'
   - title: 📦 Dependency updates
     label: 'dependencies'
+autolabeler:
+  - label: 'breaking change'
+    title:
+      - '/^break(\(.+\))?\:/i'
```

bug: The pattern for this should be looking for an [exclamation mark](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with--to-draw-attention-to-breaking-change), I think you want:
```suggestion
      - '/^[a-z]+(\(.+\))?!\:/i'
```

---

They also use `BREAKING CHANGE:` as convention, that's why I usually used `break` as our convention.

> BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with [MAJOR](http://semver.org/#summary) in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.

---

Yes but thats a footer, not a title item and its there to provide the context for the break which is typically used to describe the way to update. I would suggest sticking to the standard for `!` and ensuring that we also detail with the footer.

I'm not a fan of using a dedicated type, it's not compliant with the spec and as you loose the additional content so you don't know if it was a fix or feature etc that resulted in the breaking change.

---

You're totally right, I did not read the `footer` word in the spec 🤦 

Let's make it 100% compliant with the spec for breaking changes then. Will update the regex then